### PR TITLE
add redirect url for terraform-boot

### LINKED ISSUE #714: terraform-boot - Enable conditional oauth authentication
Since terraform-boot is a generic solution to run terraform command behind a REST API, we must implement authentication which can only be conditionally enabled. By default no authentication is necessary. 

1. Add spring-security (similar to xpanse APIs) to terraform-boot api and swagger-ui
2. Add documentation on how to enable it and what properties must be passed if enabled. 
3. Add configuration to Zitadel terraform scripts to ensure this new API client is also added to the Oauth provider.

### PR DESCRIPTION
**Pull request by jinyangyang222**
fixes eclipse-xpanse/xpanse#714

[
0905-1zitadel
](url)
[
0905-2zitadel
](url)

### CODE REVIEW

File: zitadel/terraform/environments/environment.tfvars [hunk 1]
```diff
@@ -1,2 +1,3 @@
 xpanse-ui_base_uri = "http://localhost:3000"
```
Comment (swaroopar) [inline zitadel/terraform/environments/environment.tfvars]: @jinyangyang222 i just realized that we do not need this extra environment.tfvars file. We need only local.tfvars and testbed.tfvars. 
the current values in  environment.tfvars can be moved to local.tfvars
and to the testbed.tfvars, we can add the following "https://demo.xpanse.site" value for both URIs for now. 

Please also remove the references to "environment.tfvars" file in readme.

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: zitadel/terraform/environments/environment.tfvars (hunk 1 above)
Comment (jinyangyang222) [reply zitadel/terraform/environments/environment.tfvars]: updated.

File: zitadel/terraform/terraform.tfvars [hunk 2]
```diff
@@ -10,6 +10,5 @@ post_logout_redirect_uris   = ["",
 								"/services",
 								"/myServices",
 								"/order",
-								"/healthCheck",
```
Comment (swaroopar) [inline zitadel/terraform/terraform.tfvars]: @jinyangyang222 we need this still.

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: zitadel/terraform/terraform.tfvars (hunk 2 above)
Comment (jinyangyang222) [reply zitadel/terraform/terraform.tfvars]: fixed

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**